tests/translator/output/aws-cn/function_with_custom_codedeploy_deployment_preference.json

```diff
@@ -0,0 +1,142 @@
+{
+    "Resources": {
+      "MinimalFunctionDeploymentGroup": {
+        "Type": "AWS::CodeDeploy::DeploymentGroup", 
+        "Properties": {
+          "ApplicationName": {
+            "Ref": "ServerlessDeploymentApplication"
+          }, 
+          "AutoRollbackConfiguration": {
+            "Enabled": true, 
+            "Events": [
+              "DEPLOYMENT_FAILURE", 
+              "DEPLOYMENT_STOP_ON_ALARM", 
+              "DEPLOYMENT_STOP_ON_REQUEST"
+            ]
+          }, 
+          "ServiceRoleArn": {
+            "Fn::GetAtt": [
+              "CodeDeployServiceRole", 
+              "Arn"
+            ]
+          }, 
+          "DeploymentConfigName": "TestDeploymentConfiguration", 
+          "DeploymentStyle": {
+            "DeploymentType": "BLUE_GREEN", 
+            "DeploymentOption": "WITH_TRAFFIC_CONTROL"
+          }
+        }
+      }, 
+      "MinimalFunctionRole": {
+        "Type": "AWS::IAM::Role", 
+        "Properties": {
+          "ManagedPolicyArns": [
+            "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
```

Comment:
Did `make pr` pass? ARNs in the CN partitions should be of the form `arn:aws-cn:iam:...`. Similarly in other partitions, e.g., GovCloud, there's ARNs have a different partition value. `make pr` will show you these errors so you can correct them. The Travis build also runs these tests and that has to pass in order for the PR to be merged.

Reply:
I see what's happening now. The new test input templates you've added are not getting run as part of the unit tests. You need to add them [here](https://github.com/Buffer0x7cd/serverless-application-model/blob/develop/tests/translator/test_translator.py#L210) in order to actually run them during the tests. Then run `make pr` and fix any errors.

Reply:
@jlhood Is there are any other steps involved for adding the new end to end test templates ?. I added my test templates in the location mentioned above and also fixed the ARN schema for China and Us Gov Cloud output templates, but I am receiving a strange error regarding when running make pr. 
![Screenshot from 2019-04-09 01-43-05](https://user-images.githubusercontent.com/16363909/55753810-d9483f00-5a68-11e9-87fd-b3d3c30c2038.png)
I have attached the relevant section's output in the above image. as the test_translatior.py codebase is new for me, I am thinking that I might be missing some necessary steps. please let me know your thoughts about this?

Reply:
`patching` isn't in scope inside the `except` block. You can move `try/except` to be inside the iterator.

Reply:
I think I just fixed the issue with these tests.

Reply:
Nice! Thanks for fixing those, @keetonian. Really excited to get this one merged. 🙏

Reply:
@keetonian  @jlhood apologies for not able to dedicate time for this issue through the end. can you please let me know, what was the issue with the test?

Reply:
@Buffer0x7cd These were the changes @keetonian made to enable the two tests and address the failures:

https://github.com/awslabs/serverless-application-model/pull/848/commits/0b67e7aca16a0bd074f82d4c4dcff64a1129849f

Reply:
It was mostly the partition names: China uses aws-cn and GovCloud uses aws-us-gov. Otherwise, things were pretty well correct.